dashboard/test/models/lesson_test.rb

```diff
@@ -172,7 +172,7 @@ class LessonTest < ActiveSupport::TestCase
     create :script_level, script: script, lesson: lesson2
 
     assert_match (/\/s\/bogus-script-\d+\/lessons\/2\/levels\/1/), lesson1.next_level_path_for_lesson_extras(@student)
-    assert_equal '/', lesson2.next_level_path_for_lesson_extras(@student)
+    assert_equal "/s/#{script.name}", lesson2.next_level_path_for_lesson_extras(@student)
```

Comment:
There's a not above about needing to change `LessonExtras`. The text with and without this change is "View the Unit", so this is actually an improvement. :)